src/functionalTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/BaseFunctionalTest.java

```diff
@@ -26,7 +26,7 @@
     protected String camundaUrl;
 
     @Before
-    public void setUp() {
+    public void setUp() throws Exception {
         RestAssured.baseURI = testUrl;
         RestAssured.useRelaxedHTTPSValidation();
```

Comment:
One improvement for this would be to use the RestApi actions like on the wa-task-management.

Reply:
I could. There are some discrepancies, though. In task-management you use RestAssured builder and the given from the RestAssured directly. Whereas in the task-configuration we use it from SerenityRest. In any case, I could use your approach but maybe not needed right now as there is only one single functional tests.